config/gulp/release.js

```diff
@@ -0,0 +1,149 @@
+var gulp = require('gulp');
+var dutil = require('./doc-util');
+var spawn = require('cross-spawn');
+var del = require('del');
+var runSequence = require('run-sequence');
+var merge = require('merge-stream');
+
+var streamExtensions = [
+  'omnigraffle',
+  'eps',
+  'ai',
+  'sketch',
+];
+
+function createZipArchive (taskName, src, dest, done) {
+
+  var zip = spawn('zip', [
+    '--log-info',
+    '-r',
+    src,
+    dest,
+    '-x "*.DS_Store"',
+  ]);
+
+  zip.stdout.on('data', function (data) {
+
+    if (/[\w\d]+/.test(data)) {
+
+      dutil.logData(taskName, data);
+
+    }
+
+  });
+
+  zip.stderr.on('data', function (data) {
+
+    dutil.logError(taskName, data);
+
+  });
+
+  zip.on('error', function (error) {
+
+    dutil.logError(taskName, 'Failed to create a zip archive');
+
+    done(error);
+
+  });
+
+  zip.on('close', function (code) { if (0 === code) { done(); } });
+
+}
+
+function getAssetDirectory (ext) {
+  return dutil.dirName + '-' + ext;
+}
+
+gulp.task('clean:assets:directories', function () {
+
+  dutil.logMessage('clean:assets:directories', 'Deleting temporary assets directories.');
+
+  return del([
+    dutil.pkg.name + '*',
+    '!' + dutil.pkg.name + '*.zip',
+  ]);
+
+});
+
+gulp.task('clean:assets:archives', function () {
+
+  dutil.logMessage('clean:assets:archives', 'Deleting assets zip archives.');
+
+  return del([
+    dutil.pkg.name + '*.zip',
+  ]);
+
+});
+
+gulp.task('release:process', function () {
```

Comment:
@yozlet Sure! The `release:process` task takes each type of file from the given directories / extensions and splits them out into separate `gulp#streams`. This is necessary because the source directories all contain every file in a particular folder, [but for the upcoming version of the `uswds` documentation site a separate ZIP archive is referenced for each `filetype`](https://standards-staging-new-design.apps.cloud.gov/download/). So, this task merges all these source streams and generates a folder for each `extension` type. See [this line](https://github.com/18F/web-design-standards-assets/pull/28/files#diff-09fe040bc48f79929c344efe0d53508fR5) for the start of the file extensions that are supported by this task.

The zip files and folders generated by this build process look like this:

![screen shot 2016-06-28 at 2 18 26 pm](https://cloud.githubusercontent.com/assets/706004/16427417/3b73b41e-3d3b-11e6-8e97-ed747fef0430.png)


_On a related note, I think it makes more sense to comment these `gulp.task` calls a bit better and I'll update the PR._